lib/appbundler/cli.rb

```diff
@@ -9,10 +9,12 @@ class CLI
     banner(<<-BANNER)
 * appbundler #{VERSION} *
 
-Usage: appbundler APPLICATION_DIR BINSTUB_DIR
+Usage: appbundler BUNDLE_DIR BINSTUB_DIR [GEM_NAME] [GEM_NAME] ...
```

Comment:
Yeah, I was just thinking that backcompat on this one might be tricky. If we are breaking back-compat, can we change over to proper flags?  I've been meaning to do so forever but never got around to it.  Having `--bundle_dir` and `--binstub_dir` can solve a lot of our future issues.  I dislike having to specify 2 directories and an arbitrary number of gems.